### DISCUSSION TITLE
The `ping` with `sn` is stopped?

### DISCUSSION THREAD

**Discussion by streetycat** [buckyos/CYFS#249] [Q&A]
I found that my `OOD` has been offline.

After diagnosis, I found that there is no `Ping` package to `sn`, I don't known why the `ping` is abort.

The gateway has running 6 days, the following log is the earliest that I can find:
[gateway_bdt_1727_r00023.log](https://github.com/buckyos/CYFS/files/11394774/gateway_bdt_1727_r00023.log)

The following log is the latest:
[gateway_bdt_1727_rCURRENT.log](https://github.com/buckyos/CYFS/files/11394797/gateway_bdt_1727_rCURRENT.log)

**Comment by lurenpluto** [comment]
bdt now sn ping logic is somewhat complex, especially in some network jitter, switching, I have had several similar problems before, need to locate the problem

**Comment by jing-git** [comment]
> I found that my `OOD` has been offline.
> 
> After diagnosis, I found that there is no `Ping` package to `sn`, I don't known why the `ping` is abort.
> 
> The gateway has running 6 days, the following log is the earliest that I can find: [gateway_bdt_1727_r00023.log](https://github.com/buckyos/CYFS/files/11394774/gateway_bdt_1727_r00023.log)
> 
> The following log is the latest: [gateway_bdt_1727_rCURRENT.log](https://github.com/buckyos/CYFS/files/11394797/gateway_bdt_1727_rCURRENT.log)

These 4.30 logs don't have log information for sn, are there some log files for subsequent dates?

**Reply by streetycat** [reply]
The initial log at startup has been lost, the following logs are similar as [gateway_bdt_1727_r00023.log](https://github.com/buckyos/CYFS/files/11394774/gateway_bdt_1727_r00023.log).

And there are many connections from the browser until it in the [latest log](https://github.com/buckyos/CYFS/files/11394797/gateway_bdt_1727_rCURRENT.log)

There is another clue in the `runtime` log as follow, There is no attempt to connect the `OOD` with the `Device` object from `NOC`, it's depend on the responce from `SN`?

[cyfs-runtime_bdt_43820_rCURRENT.log](https://github.com/buckyos/CYFS/files/11404944/cyfs-runtime_bdt_43820_rCURRENT.log)
